### ISSUE TITLE
Revise the logo so that the container fades more smoothly into the header container and the "PGP" is more visible. 

### ISSUE THREAD

**Issue by gissoo** [Princeton-CDH/geniza#601]

**Comment by gissoo** [comment]
reviewed with MR and RR –  MR liked the way I adjusted the tear in PGP to make PGP more visible. It turned out to be very challenging to create a smooth fade from the logo to the header in the dark mode logo, after our conversation I ended up trying a different technique, and it looks a lot better now (therefore increased the estimate from 1 to 3). We also adjusted a few things in real time (the opacity of the green and the pink text). Sent the new version for the dark mode to MR, once I get the okay then I'll close this issue and upload the logo!!! ![Screen Shot 2022-02-17 at 4.50.50 PM.png](https://images.zenhubusercontent.com/5bf307a69d595b459ec91bb1/f3722741-fd0a-41db-838a-686541ab3b78)
